DOCUMENTATION.md

```diff
@@ -87,72 +82,64 @@ Find files for manage them, and you can pipe them also.
 
 /**
  * 1st param required: array of directories
- * 2nd param not-required defualt null: pattern
- * 3th param not-required default true: boolean for recursion
+ * 2nd param not-required default '': pattern
+ * 3th param not-required default false: boolean for recursion
```

Comment:
Should be `true`. Take a look in https://github.com/reisraff/phulp/pull/14/files#diff-389a38f7e57a5cccdb6a170420f68bc0R68